.github/containerscan/allowedlist.yaml

```diff
@@ -1,6 +1,11 @@
 general:
   vulnerabilities:
     #besu-all-in-one
+    - CVE-2022-37734
+    - CVE-2022-25857
 
-    -CVE-2022-37734
-    -CVE-2022-25857
+    #cactus-plugin-ledger-connector-besu
+    - CVE-2022-24434
+    - CVE-2022-24999 #express
+    - CVE-2022-24999 #qs
+    - CVE-2022-2421
```

Comment:
@aldousalvarez I'd prefer to leave this out, unless there's a specific reason to ignore these CVEs.

Reply:
Noted on this, thank you. I already fixed it.